web/locales/en/messages.ftl

```diff
@@ -1725,8 +1725,7 @@ single-sentence = Single Sentence
 bulk-sentences = Bulk Sentences
 
 # REVIEW PAGE
-sc-review-instruction-first-part = Check
-sc-review-instruction-second-part = is this a linguistically correct sentence?
+sc-review-instruction = Check <icon></icon> is this a linguistically correct sentence?
```

Comment:
Maybe add a comment explaining to translators what `<icon></icon>` will be filled with.